app/views/insured/fdsh_ridp_verifications/_insured_ridp_modal.html.erb

```diff
@@ -1,35 +1,94 @@
+          </div>
+          <div class="col-auto px-0">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <div class="close-icon icon align-self-start m-0" aria-hidden="true">&nbsp;</div>
+            </button>
+          </div>
+        </div>
+        <div class="modal-body px-0 pt-0">
+          <label class="weight-n mb-2"><%= l10n("insured.consumer_roles.upload_ridp_documents.document_info") %></label>
+          <div>
+            <%= form_tag insured_ridp_documents_upload_path, multipart: true do %>
               <fieldset>
-                <legend for="ridp_subject"><%= l10n("insured.consumer_roles.upload_ridp_documents.id_type") %></legend>
+                <legend><%= l10n("insured.consumer_roles.upload_ridp_documents.document_type") %></legend>
                 <span>
-                  <label class="mb-4" for="ridp_subject_Driver_s_License_issued_by_state_or_territory"><%= radio_button_tag 'ridp_subject', "Driver's License issued by state or territory", true %><%= l10n("insured.consumer_roles.upload_ridp_documents.driver_license") %></label>
-                  <label class="mb-4" for="ridp_subject_School_identification_card"><%= radio_button_tag 'ridp_subject', 'School identification card', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.school_id") %></label>
-                  <label class="mb-4" for="ridp_subject_U.S._military_card_or_draft_record"><%= radio_button_tag 'ridp_subject', 'U.S. military card or draft record', false%><%= l10n("insured.consumer_roles.upload_ridp_documents.military_id") %></label>
-                  <label class="mb-4" for="ridp_subject_Identification_card_issued_by_the_federal__state_or_local_government__including_a_U.S._Passport"><%= radio_button_tag 'ridp_subject', 'Identification card issued by the federal, state or local government, including a U.S. Passport', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.passport") %></label>
-                  <label class="mb-4" for="ridp_subject_Military_dependent_s_identification_card"><%= radio_button_tag 'ridp_subject', "Military dependent's identification card", false %><%= l10n("insured.consumer_roles.upload_ridp_documents.military_dependent") %></label>
-                  <label class="mb-4" for="ridp_subject_Native_American_tribal_document"><%= radio_button_tag 'ridp_subject', 'Native American tribal document', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.native_american") %></label>
-                  <label class="mb-4" for="ridp_subject_U.S._Coast_Guard_Merchant_Mariner_card"><%= radio_button_tag 'ridp_subject', 'U.S. Coast Guard Merchant Mariner card', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.coast_guard") %></label>
+                  <% options = {
+                    :driver_license => "Driver's License issued by state or territory", 
```

Comment:
I think these need to be added to translations, additionally maybe this options hash should be constructed in a helper instead of the view

Reply:
The values in the hash are used for the form value used on submit, not user-facing content. The keys are used to get the translations from the translation file.

Definitely open to move this to a helper, in my mind though this is a simple declaration without any logic in it. Do you think it still makes sense?

Reply:
The strings are used for the submit value, the keys are used to construct the translations